docs/releasenotes/release-1.125.md

```diff
@@ -2,42 +2,61 @@
 
 * Special shout-outs to @600lyy, @acpana, @anhdle-sso, @cheftako, @ericpang777, @gemmahou, @hankfreund, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nb-goog, @svetakvsundhar, @xiaoweim, @yuwenma, @zicongmei, @ziyue-101 for their contributions to this release.
 
-## Announcement 
-
 ## New Beta Resources (Direct Reconciler):
 
-* `BigQueryAnlayticsHubDataExchange`
-* `PrivilegedAccessManagerEntitlement`
-* `RedisCluster`
-* `WorkstationCluster`
+* [`BigQueryConnectionConnection`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryconnection/bigqueryconnectionconnection)
+
+     * Manage [connections](https://cloud.google.com/bigquery/docs/working-with-connections) to connect to Google services and external data sources
+
+* [`BigQueryAnalyticsHubDataExchange`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryanalyticshub/bigqueryanalyticshubdataexchange.md)
+
+  * Manage [data exchange](https://cloud.google.com/bigquery/docs/analytics-hub-introduction#data_exchanges) to enable self-service data sharing
+
+* [`PrivilegedAccessManagerEntitlement`](https://cloud.google.com/config-connector/docs/reference/resource-docs/privilegedaccessmanager/privilegedaccessmanagerentitlement) 
+
+     * Manage [entitlements](https://cloud.google.com/iam/docs/pam-create-entitlements) to grant for projects, folders, and organizations
+ 
+* [`WorkstationCluster`](https://cloud.google.com/config-connector/docs/reference/resource-docs/workstations/workstationcluster.md)
+
+  * Manage [workstation cluster](https://cloud.google.com/workstations/docs/overview) to define a group of workstations in a particular region and the VPC network they're attached to.
+
+## New Alpha Resources (Direct Reconciler):
 
+* `KMSAutokeyConfig`
+ 
+    * Manage the [KMS auto key](https://cloud.google.com/kms/docs/autokey-overview) which simplifies the CMEKs provisioning and assignment. 
+ 
 ## New Fields:
 
-* `AlloyDBInstance`
+* [`AlloyDBInstance`](https://cloud.google.com/config-connector/docs/reference/resource-docs/alloydb/alloydbinstance) (Beta)
   * Added `spec.networkConfig.enableOutboundPublicIp` field.
   * Added `status.outboundPublicIpAddresses` field.
 
-## Modified Beta Reconciliation
+## Reconciliation Improvements
 
 We migrated the following resources from the Terraform-based or DCL-based controller to the new Direct Controller to enhance the reliability and performance. The resource CRD is unchanged.
 
-* `SQLInstance`
+* [`SQLInstance`](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance)  
```

Comment:
@jasonvigil check my recollection. We have added support for running SQLInstance via Direct Controller but not actually migrated to it?

Reply:
That's fair. Maybe the wording should say we "enabled" the direct controller, instead of "migrated"

Reply:
Thanks! I updated the release note to reflect the discussion